plugins/woocommerce-admin/client/activity-panel/activity-panel.js

```diff
@@ -236,6 +236,13 @@ export const ActivityPanel = ( { isEmbedded, query } ) => {
 		return query.page === 'wc-admin' && ! query.path;
 	};
 
+	const isProductPage = () => {
+		return (
+			query.page === 'wc-admin' &&
+			/^(\/add-product|\/product)/gi.test( query.path )
```

Comment:
Could we use a check to see if that path is strictly equal to `product` or `add-product` instead of regex?  If we were to add a new page with a path of `product-settings`, for example, this would give us a false positive and remove the activity panel items.

Reply:
Awesome!!! what do you think if we check for `/add-product/` and `/product/` segments strictly? 
```javascript
/^\/(add-product|product\/.+)$/gi.test( query.path )
```
Since the edit product page path is something like `/product/{id}` that's why I used a regex.